torch/nn/quantized/functional.py

```diff
@@ -94,14 +94,15 @@ def conv2d(input, weight, bias,
         >>> from torch.nn.quantized import functional as qF
         >>> filters = torch.randn(8, 4, 3, 3, dtype=torch.float)
         >>> inputs = torch.randn(1, 4, 5, 5, dtype=torch.float)
-        >>> bias = torch.randn(4, dtype=torch.float)
+        >>> bias = torch.randn(8, dtype=torch.float)
         >>>
         >>> scale, zero_point = 1.0, 0
-        >>> dtype = torch.quint8
+        >>> dtype_inputs = torch.quint8
+        >>> dtype_filters = torch.qint8
```

Comment:
Can we also add a check like what we have for padding_mode so that if dtype_inputs and dtype_filters do not match what is supported (quint8 for inputs and qint8 for filters) we error out?

Reply:
Will add

Reply:
@raghuramank100 

Added warning and also fixed the same for 3D.